gui/vm/forms.py

```diff
@@ -97,13 +123,34 @@ class DeviceForm(ModelForm):
         required=False,
         dirsonly=False,
     )
+    DISK_raw_boot = forms.BooleanField(
+        label=_('Disk boot'),
+        widget=forms.widgets.HiddenInput(),
+        required=False,
+        initial=False,
+    )
+    ROOT_password = forms.CharField(
+        label=_('Password'),
+        max_length=50,
+        widget=forms.widgets.HiddenInput(),
+        required=False,
+        help_text=_("Set the user rancher password."),
+    )
     DISK_sectorsize = forms.IntegerField(
         label=_('Disk sectorsize'),
         required=False,
         initial=0,
         help_text=_("Logical and physical sector size in bytes of the emulated disk."
                     "If 0, a sector size is not set."),
     )
+    DISK_raw_size = forms.CharField(
+            label=_('Disk size'),
+            widget=forms.widgets.HiddenInput(),
+            required=False,
+            initial=0,
+            validators=[RegexValidator("^(\d*)\s?([M|G|T]?)$", "You can use M, G or T as unit size, otherwise it will use G by default.")],
+            help_text=_("You can resize a given raw disk or use 0 to use it as it is."),
```

Comment:
Avoid using "you".

"Resize the existing raw disk. Enter 0 to use the disk with the current size."